From 574234227b487608d9cc89e8fb2dc8abf0fea4a8 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:22 -0400
Subject: [scsi] qla2xxx: Remove ISP_ABORT_NEEDED and ISP_ABORT_RETRY checks from watchdog function for ISP8044

Message-id: <1408391817-6389-35-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87964
O-Subject: [PATCH RHEL 7.1 e-stor 34/69] qla2xxx: Remove ISP_ABORT_NEEDED and ISP_ABORT_RETRY checks from watchdog function for ISP8044.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Atul Deshmukh <atul.deshmukh@qlogic.com>

scsi commit id 06aec108f5e2481246914347709d2d0c89569cf7

Signed-off-by: Atul Deshmukh <atul.deshmukh@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_nx2.c b/drivers/scsi/qla2xxx/qla_nx2.c
index 5f88931..abd7e05 100644
--- a/drivers/scsi/qla2xxx/qla_nx2.c
+++ b/drivers/scsi/qla2xxx/qla_nx2.c
@@ -2014,8 +2014,6 @@ qla8044_watchdog(struct scsi_qla_host *vha)
 
  /* don't poll if reset is going on or FW hang in quiescent state */
  if (!(test_bit(ABORT_ISP_ACTIVE, &vha->dpc_flags) ||
-     test_bit(ISP_ABORT_NEEDED, &vha->dpc_flags) ||
-     test_bit(ISP_ABORT_RETRY, &vha->dpc_flags) ||
      test_bit(FCOE_CTX_RESET_NEEDED, &vha->dpc_flags))) {
   dev_state = qla8044_rd_direct(vha, QLA8044_CRB_DEV_STATE_INDEX);
 
-- 
1.7.1